feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/post/PostItem.kt

```diff
@@ -119,15 +127,36 @@ internal fun PostItem(
                     it
```

Comment:
I think we should move the modifier modification outside to the function body for better readability, at the moment its quite a lot. Maybe just creating a function something like `val applyPinStatusToModifier: (Modifier) -> Modifier = { if (isPinned) ...` would be sufficient